### PR TITLE
Added change-request-reviews.md

### PR DESCRIPTION
**Pull request by zhongjun2**

### CODE REVIEW

**Comment by king-gao** [comment]
Hi @zhongjun2 , you need to check the DCO , or sign-off your commit :)

**Comment by zhongjun2** [comment]
> Hi @zhongjun2 , you need to check the DCO , or sign-off your commit :)

Thanks.  
Done

File: chinese/wg-evolution/focus-areas/code-development-process-quality/change-request-reviews.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# 变更请求评审
+
+### 这个指标是一个候选版本。要对此指标发表评论，请参阅议题 # [437](https://github.com/chaoss/wg-evolution/issues/437)。在征求意见期后，该指标将包含在下一个常规版本中。
```
Comment (eyehwan) [inline chinese/wg-evolution/focus-areas/code-development-process-quality/change-request-reviews.md:3]: not needed

File: chinese/wg-evolution/focus-areas/code-development-process-quality/change-request-reviews.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+* Kemerer、Chris F. 和 Mark C. Paulk。 “设计和代码审查对软件质量的影响：基于 psp 数据的实证研究。” _IEEE 软件工程交易_ 35.4 (2009): 534-550
+
+## 贡献者
+* Kevin Lumbard
+* Elizabeth Barron
+* Vinod Ahuja
+* Sean Goggins
+* Enoch Kaxada
+
+***该指标最后一次审核是在 2022 年 2 月 22 日，作为 2022-04 版本的一部分。***
```
Comment (eyehwan) [inline chinese/wg-evolution/focus-areas/code-development-process-quality/change-request-reviews.md:75]: not needed.

**Review by eyehwan — COMMENTED** [review]
minor comments

**Comment by eyehwan** [comment]
#116

File: chinese/wg-evolution/focus-areas/code-development-process-quality/change-request-reviews.md (hunk 1 above)
Comment (zhongjun2) [reply chinese/wg-evolution/focus-areas/code-development-process-quality/change-request-reviews.md]: done

File: chinese/wg-evolution/focus-areas/code-development-process-quality/change-request-reviews.md (hunk 2 above)
Comment (zhongjun2) [reply chinese/wg-evolution/focus-areas/code-development-process-quality/change-request-reviews.md]: done

**Pull request merged**